client/pages/form-pages/application-table.tsx

```diff
@@ -0,0 +1,151 @@
+import { Button, SearchBar } from '@components';
+import { Pagination } from '../../components/form/Pagination';
+import { useState } from 'react';
+import initialValues from './example_submission_data.json';
+
+export interface ActivitiesGapProps {
+  // step: number;
+  // title: string;
+}
+
+const TableHeader: React.FC = () => {
+  //   const { initialValues } = usePlanningActivitiesGap();
+  const headers = [
+    'Confirmation ID',
+    'Facility',
+    'Project',
+    'Estimated Cost',
+    'Asks',
+    'Assigned to',
+    'Last update',
+    'Status',
+  ];
+  const tdStyles =
+    'table-td table-header px-6 py-4 text-left text-sm font-strong text-bcBluePrimary border-b-2  border-amber-200';
+  return (
+    <thead className='border-b bg-gray-50 table-row-fixed table-header '>
+      <tr>
+        {headers &&
+          headers.map((title: string, index: number) => (
+            <th key={`th${index}`} className={tdStyles}>
+              {title}
+            </th>
+          ))}
+      </tr>
+    </thead>
+  );
+};
+
+const TableBody: React.FC = (applications: any) => {
+  const handleSelectRow = (index: number) => {
+    alert(index);
+  };
+  const tdStyles =
+    'table-td px-6 py-4 text-left text-sm font-strong text-bcBluePrimary flexitems-center justify-between';
+  return (
+    <tbody>
+      {applications.applications &&
+        applications.applications.map((row: any, index: number) => (
+          <>
+            <tr
+              key={`row${index}`}
+              onClick={() => handleSelectRow(row.BCAAP_Form_ID)}
+              className='bg-white border-b table-row-fixed'
+            >
+              <td className={tdStyles}>{row.Confirmation_ID}</td>
```

Comment:
Could you also make sure that the styling, colors align with the design!

Reply:
OK